Math Go/Controller/ViewController.swift

```diff
@@ -6,14 +6,58 @@
 //
 
 import UIKit
+import MapKit
+import CoreLocation
 
+/* Notifies the locationManager when it has updated the position of the device
+Source: https://www.raywenderlich.com/764-augmented-reality-ios-tutorial-location-based */
 class ViewController: UIViewController {
```

Comment:
Since we will have several views (Map, Catch, Inventory, Profile), can you move Map view specific code into a file named "MapViewController"?